docs/fundamentals/flow/flow-api.md

```diff
@@ -107,73 +126,16 @@ with f:
     print(bar_response_docs.texts)
 ```
 
-This will print
+This will print:
 
 ```text
 ['foo was called']
 ['bar was called']
 ```
 
-The `Client` also offers the convenience functions `.search()` and  `.index()`, which are just shortcuts for `.post(on='/search')` and `.post(on='/index')`.
-
-## Limiting outstanding requests
```

Comment:
it should be in the Client API right ? https://docs.jina.ai/fundamentals/flow/client/

Reply:
The note about the client? That should be in the client section. This page is about Flow API overall + using it as a service, we think

Reply:
the limiting outstandinf requests

Reply:
Yes this will go somewhere else. We are restructuring this, because stuff like that is not part of defining the Flow API, but rather how you interact with that API once it is up. So this will go in the Client section, or a different section altogether.